app/views/layouts/_topnav.html.haml

```diff
@@ -59,3 +33,17 @@
           = link_to(t(:_help), "#{$WIKI_HELP_PAGE}", target: "_blank", class: "dropdown-item")
           = link_to(t(:_release_notes), "#{$RELEASE_NOTES}", target: "_blank", class: "dropdown-item")
           = link_to(t(:_publications), $PUBLICATIONS_URL, target: "_blank", class: "dropdown-item")
+:javascript
```

Comment:
instead of using this js code, use directly bootstrap if possible or a stimulus controller (or add it to an existent old js file )

Reply:
It's not possible to use bootstrap with the existing code,
It's better to keep the existing js lines like this, cause It uses querySelectorAll with 8 classes, adding it as targets in stimulus will just complicate things.